sequencer/api/migrations/postgres/V601__stake_table_events_table.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE stake_table_events (
+  id INTEGER DEFAULT 0 PRIMARY KEY,
```

Comment:
Since the plan is for this table to only have one row how about we enforce it via the schema?